### ISSUE TITLE
Unable to use btcli on 8.1.0, unable to access wallet

### ISSUE THREAD

**Issue by stong** [camfairchild/bittensor#2338]
## Description

On a fresh install of 8.1.0 from PyPI, the user is unable to access their wallet due to an AttributeError.

## Workaround

Downgrade to 8.0.0.

`pip install --upgrade bittensor[btcli]==8.0.0`

## Stack trace

```
btcli wallet balance --all
Enter the wallet path (Hint: You can set this with `btcli config set --wallet-path`) [~/.bittensor/wallets/]:
⠋ Connecting to Substrate: Network: finney, Chain: wss://entrypoint-finney.opentensor.
╭─────────────────────────────────────────────────────────────────────────────── Traceback (most recent call last) ────────────────────────────────────────────────────────────────────────────────╮
│ /mamba/envs/bittensor/lib/python3.10/site-packages/bittensor_cli/cli.py:2015 in wallet_balance                                                                                                   │
│                                                                                                                                                                                                  │
│   2012 │   │   │   wallet_name, wallet_path, wallet_hotkey, ask_for=ask_for, validate=validate                                                                                                   │
│   2013 │   │   )                                                                                                                                                                                 │
│   2014 │   │   subtensor = self.initialize_chain(network)                                                                                                                                        │
│ ❱ 2015 │   │   return self._run_command(                                                                                                                                                         │
│   2016 │   │   │   wallets.wallet_balance(wallet, subtensor, all_balances)                                                                                                                       │
│   2017 │   │   )                                                                                                                                                                                 │
│   2018                                                                                                                                                                                           │
│                                                                                                                                                                                                  │
│ ╭─────────────────────────────────────────────────── locals ────────────────────────────────────────────────────╮                                                                                │
│ │  all_balances = True                                                                                          │                                                                                │
│ │       ask_for = [<WalletOptions.PATH: 'path'>]                                                                │                                                                                │
│ │       network = None                                                                                          │                                                                                │
│ │         quiet = False                                                                                         │                                                                                │
│ │          self = <bittensor_cli.cli.CLIManager object at 0x7f5683533070>                                       │                                                                                │
│ │     subtensor = <bittensor_cli.src.bittensor.subtensor_interface.SubtensorInterface object at 0x7f565f3dd5d0> │                                                                                │
│ │      validate = <WalletValidationTypes.NONE: None>                                                            │                                                                                │
│ │       verbose = False                                                                                         │                                                                                │
│ │        wallet = name: 'default', hotkey: 'default', path: '~/xxxxx/.bittensor/wallets/'                       │                                                                                │
│ │ wallet_hotkey = None                                                                                          │                                                                                │
│ │   wallet_name = None                                                                                          │                                                                                │
│ │   wallet_path = None                                                                                          │                                                                                │
│ ╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────╯                                                                                │
│                                                                                                                                                                                                  │
│ /mamba/envs/bittensor/lib/python3.10/site-packages/bittensor_cli/cli.py:793 in _run_command                                                                                                      │
│                                                                                                                                                                                                  │
│    790 │   │   │   return asyncio.get_event_loop().run_until_complete(_run())                   ╭──────────────────────────── locals ────────────────────────────╮                               │
│    791 │   │   else:                                                                            │  cmd = <coroutine object wallet_balance at 0x7f565f464ac0>     │                               │
│    792 │   │   │   # For Python 3.10 or higher                                                  │ self = <bittensor_cli.cli.CLIManager object at 0x7f5683533070> │                               │
│ ❱  793 │   │   │   return asyncio.run(_run())                                                   ╰────────────────────────────────────────────────────────────────╯                               │
│    794 │                                                                                                                                                                                         │
│    795 │   def main_callback(                                                                                                                                                                    │
│    796 │   │   self,                                                                                                                                                                             │
│                                                                                                                                                                                                  │
│ /mamba/envs/bittensor/lib/python3.10/asyncio/runners.py:44 in run                                                                                                                                │
│                                                                                                                                                                                                  │
│   41 │   │   events.set_event_loop(loop)                                                      ╭────────────────────────────────────── locals ──────────────────────────────────────╮             │
│   42 │   │   if debug is not None:                                                            │ debug = None                                                                       │             │
│   43 │   │   │   loop.set_debug(debug)                                                        │  loop = <_UnixSelectorEventLoop running=False closed=True debug=False>             │             │
│ ❱ 44 │   │   return loop.run_until_complete(main)                                             │  main = <coroutine object CLIManager._run_command.<locals>._run at 0x7f565f464c10> │             │
│   45 │   finally:                                                                             ╰────────────────────────────────────────────────────────────────────────────────────╯             │
│   46 │   │   try:                                                                                                                                                                                │
│   47 │   │   │   _cancel_all_tasks(loop)                                                                                                                                                         │
│                                                                                                                                                                                                  │
│ /mamba/envs/bittensor/lib/python3.10/asyncio/base_events.py:641 in run_until_complete
│
│    232 │   with console.status("Retrieving balances", spinner="aesthetic") as status:
│    233 │   │   if all_balances:
│    234 │   │   │   print_verbose("Fetching data for all wallets", status)
│ ❱  235 │   │   │   coldkeys, wallet_names = _get_coldkey_ss58_addresses_for_path(wallet.path)
│    236 │   │   else:
│    237 │   │   │   print_verbose(f"Fetching data for wallet: {wallet.name}", status)
│    238 │   │   │   coldkeys = [wallet.coldkeypub.ss58_address]
│
│ ╭─────────────────────────────────────────────────── locals ───────────────────────────────────────────────────╮
│ │ all_balances = True                                                                                          │
│ │       status = <rich.status.Status object at 0x7f565f396c20>                                                 │
│ │    subtensor = <bittensor_cli.src.bittensor.subtensor_interface.SubtensorInterface object at 0x7f565f3dd5d0> │
│ │       wallet = name: 'default', hotkey: 'default', path: '~/xxxxx/.bittensor/wallets/'                       │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
│
│ /mamba/envs/bittensor/lib/python3.10/site-packages/bittensor_cli/src/commands/wallets.py:216 in _get_coldkey_ss58_addresses_for_path
│
│    213 │   │   for wallet in wallets                                                            ╭──────────────────────────────── locals ─────────────────────────────────╮
│    214 │   │   if os.path.exists(os.path.join(abs_path, wallet, "coldkeypub.txt"))              │      abs_path = 'xxxxx/.bittensor/wallets'                              │
│    215 │   ]                                                                                    │ coldkey_paths = [                                                       │
│ ❱  216 │   ss58_addresses = [Keyfile(path).keypair.ss58_address for path in coldkey_paths]      │                 │   'xxxxx/.bittensor/wallets/new/coldkeypub.txt',      │
│    217 │                                                                                        │                 │   'xxxxx/.bittensor/wallets/default2/coldkeypub.txt', │
│    218 │   return ss58_addresses, [                                                             │                 │   'xxxxx/.bittensor/wallets/default3/coldkeypub.txt', │
│    219 │   │   os.path.basename(os.path.dirname(path)) for path in coldkey_paths                │                 │   'xxxxx/.bittensor/wallets/default/coldkeypub.txt'   │
│                                                                                                 │                 ]                                                       │
│                                                                                                 │          path = 'xxxxx/.bittensor/wallets/'                             │
│                                                                                                 │       wallets = ['new', 'default2', 'default3', 'default']              │
│                                                                                                 ╰─────────────────────────────────────────────────────────────────────────╯
│
│ /mamba/envs/bittensor/lib/python3.10/site-packages/bittensor_cli/src/commands/wallets.py:216 in <listcomp>
│
│    213 │   │   for wallet in wallets                                                            ╭─────────────────────── locals ───────────────────────╮
│    214 │   │   if os.path.exists(os.path.join(abs_path, wallet, "coldkeypub.txt"))              │   .0 = <list_iterator object at 0x7f565f3de1a0>      │
│    215 │   ]                                                                                    │ path = 'xxxxx/.bittensor/wallets/new/coldkeypub.txt' │
│ ❱  216 │   ss58_addresses = [Keyfile(path).keypair.ss58_address for path in coldkey_paths]      ╰──────────────────────────────────────────────────────╯
│    217 │
│    218 │   return ss58_addresses, [
│    219 │   │   os.path.basename(os.path.dirname(path)) for path in coldkey_paths
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
AttributeError: 'builtins.Keyfile' object has no attribute 'keypair'
```

### To Reproduce

```
mamba create -n bittensor
mamba activate bittensor
mamba install python==3.10
pip install bittensor[btcli]
btcli wallet balance --all
```

### Expected behavior

The user is able to access their wallet

### Environment

Ubuntu 22.04 LTS

**Comment by rajkaramchedu** [comment]
Thanks! The team is on it!

**Comment by ibraheem-abe** [comment]
Fixed in `v8.1.1` of btcli & sdk and `v2.0.1` of btwallet. Thanks for reporting!
